### ISSUE TITLE
Siri Problem

### ISSUE THREAD

**Issue by james-ico** [checkra1n/BugTracker#757]
**Tell us about your setup:**
1. What iDevice are you using? iPod touch 6th Generation
2. On what version of iOS is it? 12.4.4
3. What version of checkra1n are you using? GUI or CLI? 0.9.7
4. What is your host system (OS version? Hackintosh? VM? etc.)? High Sierra
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? Apple cable

Guys how can u fix that siri issue ? When I open siri, there is no way to talk with her. When I touch to talk button on siri, there is no action. 
I uploaded it : https://streamable.com/ca9xk

**Comment by computersteve** [comment]
Yeah this is happening also on the iPhone X with 13.3.1

**Comment by james-ico** [comment]
Please do something for that guys.

**Comment by Siguza** [comment]
Have you tried with Safe Mode enabled (The checkbox in the checkra1n GUI, not Cydia's Safe Mode)?

**Comment by james-ico** [comment]
I will try and I will write the result

**Comment by james-ico** [comment]
I tried still not working.

**Comment by james-ico** [comment]
> Have you tried with Safe Mode enabled (The checkbox in the checkra1n GUI, not Cydia's Safe Mode)?

I tried still not working.

**Comment by computersteve** [comment]
Yes me as well. I also tried a full restore & setup as new. Siri stops working over bluetooth with my Beats / Car.. With Checkrain active. If I just boot up normally then it works fine.

**Comment by joechiazza** [comment]
iPhone X 13.5.1 “Hey Siri” is recognizing very little volume at all from its microphone input, therefore not operational using checkra1n 0.10.2 beta. I have tested the voice recorder and microphone input is working at normal levels. “Hey Siri” no longer works at all, even after I reboot and restore to factory defaults. Any suggestions would be much appreciated.

**Comment by jleseane23** [comment]
Same here. I thought it was hardware related but hearing everyone here makes me think otherwise. Mics work in everything else. Just not Siri or dictation.

iPhone X
Was on 13.6.1 when having the issues (troubleshooting made me upgrade to 13.7)
Used Checkra1n in Linux using CLI
Used a apple certified USB to Lightning cable
